src/procedure_manager.rs

```diff
@@ -46,22 +48,23 @@ pub fn run_project_procedure(project: &Project, branch: &Branch, procedure: &Pro
 
             // Print stdout and stderr from child process asynchronously
             let pname: String = procedure_name.clone();
+            let plog: Option<String> = procedure_log.clone();
```

Comment:
Still kind of confused why we clone this. The log runtime should take ownership?

Reply:
I am not sure if this would compile without a clone here

Reply:
Ah yes. Because it is in a loop, the value moves in to the void then needs to be moved after its destruction.

Reply:
Oh yes